NineChronicles.Headless/Properties/GrpcRateLimitOptions.cs

```diff
@@ -0,0 +1,11 @@
+namespace NineChronicles.Headless.Properties
+{
+    public class GrpcRateLimitOptions
+    {
+        public const string GrpcRateLimit = "GrpcRateLimit";
```

Comment:
Where is this constant used?

Reply:
It's actually not used so I'll delete it. 👍